src/cmds/testing/fb_draw_image/fb_draw_image.c

```diff
@@ -0,0 +1,115 @@
+/**
+ * @file
+ * @brief app for draw images to the framebuffer
+ *
+ * @date July 14, 2019
+ * @author Filipp Chubukov
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <inttypes.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/ioctl.h>
+#include <fs/file_format.h>
+#include <lib/bmp.h>
+
+#include <drivers/video/fb.h>
+
+static void print_usage(void) {
+	printf(
+		"Usage : fb_draw_image [FILE]\n"
+		"If [FILE] format is supporting then draw it to the framebuffer\n"
+	 );
+}
+
+static int get_fbp(uint8_t *fbp, struct fb_info *fb_info, uint8_t *img, uint32_t width, uint32_t height) {
+	int x, y, pos = 0;
+	long int idx = fb_info->var.xoffset + fb_info->var.xres * fb_info->var.yoffset;
+	int bytes_per_pixel = fb_info->var.bits_per_pixel / 8; /* Figure out the size of the screen in bytes */
```

Comment:
I think this comment is redundant (name of the variable is pretty self-explanatory)

Reply:
remove